engines/python/src/main/java/ai/djl/python/engine/RollingBatch.java

```diff
@@ -48,11 +48,13 @@ class RollingBatch implements Runnable {
     private ReentrantLock lock;
     private Condition canAdd;
     private Condition canRead;
+    private boolean enableStreaming;
 
-    RollingBatch(PyProcess process, int maxRollingBatchSize, int timeout) {
+    RollingBatch(PyProcess process, int maxRollingBatchSize, int timeout, boolean enableStreaming) {
```

Comment:
In which case we want to disable streaming if rolling batch is enabled?

Reply:
Some cases customer just want to get output in 1 go. Also for short sequence generation, not necessary for rolling batch